src/components/theme/__tests__/theme.test.js

```diff
@@ -0,0 +1,164 @@
+import merge from 'lodash/merge';
+import Color from 'color';
+import randomSwatches from '../../../../test/fixtures/theme/colors';
+import { theme } from '../theme';
+import { generateContrastColors, MIN_CONTRAST } from '../contrast-colors';
+import * as vars from '../../../styles/variables';
+
+describe('Validate generated color accessibility', () => {
+    const accessibleKeys = ['backgroundHover', 'backgroundActive', 'borderHover', 'borderActive'];
+    const contrastKey = 'foreground';
+
+    // Potentially problematic colors merged with random 1000 swatches from fixture set.
+    const colorCases = ['#ffffff', '#000000', '#232300', '#c50213', ...randomSwatches];
+
+    test.each(colorCases)('given %p as argument, returns accessible color contrast', colorKey => {
+        const dynamicTheme = generateContrastColors(colorKey);
+
+        accessibleKeys.forEach(key => {
+            expect(
+                Color(dynamicTheme.primary[key]).contrast(Color(dynamicTheme.primary[contrastKey])),
+            ).toBeGreaterThanOrEqual(MIN_CONTRAST);
+        });
+    });
+});
+
+test('should contain default Box theme', () => {
+    expect(theme).toMatchInlineSnapshot(`
+        Object {
+          "base": Object {
+            "background": "#fff",
+            "backgroundActive": "#767676",
+            "backgroundHover": "#bcbcbc",
+            "border": "#4e4e4e",
+            "borderActive": "#767676",
+            "borderHover": "#bcbcbc",
+            "foreground": "#222",
+          },
+          "modes": Object {
+            "admin": Object {},
+            "dark": Object {},
+          },
+          "primary": Object {
+            "background": "#0061d5",
+            "backgroundActive": "#0049A0",
+            "backgroundHover": "#006FF3",
+            "border": "#0061d5",
+            "borderActive": "#0049A0",
+            "borderHover": "#006FF3",
+            "foreground": "#fff",
+          },
+        }
+    `);
+});
+
+test('should generate box brand colors', () => {
+    const dynamicTheme = generateContrastColors(vars.bdlBoxBlue);
+
+    expect(dynamicTheme).toMatchInlineSnapshot(`
```

Comment:
I think we've been avoiding inline snapshots. Can we use a Jest matcher like `toContain`, instead?

Reply:
I'll move these over to a separate snapshot instead of inline for convention